### PR TITLE
Finishing Design for Requirements for SDS

### PR DESCRIPTION
**Pull request by JonathanTomaro**
Just the coding design

### CODE REVIEW

File: SoftwareDesignDocument..txt [hunk 1]
```diff
@@ -26,10 +26,46 @@ Code:
...
+Req 1.0.1
+	Rules for tank movement.
+Code:
+	void tankMovement()
+	     	// Speed is just for an example value
+		// Will move with same controls as character
+		int tank_velocity = 2;
+		//Block height 1 more than current level of tank automatically move on top of block
+Req 1.0.2
+	Despawn the tank after 5 collisions with projectile block.
```
Comment (samschafer77) [inline SoftwareDesignDocument..txt:38]: I think it would be easier if we just used the hp system from vanilla minecraft.

**Review by samschafer77 — COMMENTED** [review]
The only change I would make would be to change the amount of hp equal to the amount of hp a character has in minecraft, I think that would make that a lot easier. Everything else looks good.

File: SoftwareDesignDocument..txt (hunk 1 above)
Comment (JonathanTomaro) [reply SoftwareDesignDocument..txt:38]: That's a good idea, but I don't think the craft that we are using doesn't have a health function we could reuse. If I remember Minecraft has 10 hearts so I was just doing 2 hearts of damage each time if there is an health function I missed.

Comment (samschafer77) [reply SoftwareDesignDocument..txt:38]: Yeah that sounds good, I just checked and this version doesn't have the heart system like I thought it did.

**Review by samschafer77 — APPROVED** [review]
After discussing the health point changes we came to a resolution.

**Pull request merged**